src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+
+  if (days > 2 && days < 7) {
+    return total - 20;
```

Comment:
Consider using constants for the discount values (20 and 50) and for daily rental cost (40) to avoid magic numbers and improve readability.